dokan/dokan.c

```diff
@@ -249,6 +249,12 @@ int DOKANAPI DokanMain(PDOKAN_OPTIONS DokanOptions,
     return DOKAN_MOUNT_ERROR;
   }
 
+  
+  // wait until the device is realy mounted
+  if (DokanOptions->Options & DOKAN_OPTION_MOUNT_MANAGER) {
+	  WaitForMount(instance);
```

Comment:
An handle is needed which will be receiver of the messages. If Desktop handle is passed I have no option to access the received messages.


Von Samsung Mobile gesendet


-------- Ursprüngliche Nachricht --------
Von: joepperkins
Datum:15.01.2016 17:49 (GMT+01:00)
An: dokan-dev/dokany
Cc: Babic Marinko
Betreff: Re: [dokany] Fix mounting for socket connection (#140)


In dokan/dokan.c<https://github.com/dokan-dev/dokany/pull/140#discussion_r49875768>:

> @@ -249,6 +249,12 @@ int DOKANAPI DokanMain(PDOKAN_OPTIONS DokanOptions,
>      return DOKAN_MOUNT_ERROR;
>    }
>
> +
> +  // wait until the device is realy mounted
> +  if (DokanOptions->Options & DOKAN_OPTION_MOUNT_MANAGER) {
> +       WaitForMount(instance);


The getdesktopwindow should work for your handle. https://msdn.microsoft.com/en-us/library/windows/desktop/ms633504(v=vs.85).aspx Sent on a Sprint Samsung Galaxy Note® 3
-------- Original message -------- From: Marinko <notifications@github.com> Date: 01/15/2016 1:15 AM (GMT-08:00) To: dokan-dev/dokany <dokany@noreply.github.com> Subject: Re: [dokany] Fix mounting for socket connection (#140<https://github.com/dokan-dev/dokany/pull/140>) In dokan/dokan.c<#140 (comment)<https://github.com/dokan-dev/dokany/pull/140#discussion_r49833461>>:
@@ -249,6 +249,12 @@ int DOKANAPI DokanMain(PDOKAN_OPTIONS DokanOptions, return DOKAN_MOUNT_ERROR; } + + // wait until the device is realy mounted + if (DokanOptions->Options & DOKAN_OPTION_MOUNT_MANAGER) { + WaitForMount(instance);
RegisterDeviceNotification needs a window handle. It can be realized using a hidden window but it seems again to me to be a hack. - Reply to this email directly or view it on GitHub<https://github.com/dokan-dev/dokany/pull/140/files#r49833461>.

-
Reply to this email directly or view it on GitHub<https://github.com/dokan-dev/dokany/pull/140/files#r49875768>.